troubleshooting/application-title-in-email-templates-wrong.md

```diff
@@ -0,0 +1,55 @@
+---
+title: &#123;&#123;Application.Title&#125;&#125; in Email Template is Wrong
+description: You renamed the name of your project in Telerik Platform portal, but in emails &#123;&#123;Application.Title&#125;&#125; still the old name appears.
+type: troubleshooting
+page_title: How to Change the &#123;&#123;Application.Title&#125;&#125; in Email Templates
+slug: application-title-in-email-templates-wrong
+position:
+tags: backendservices, portal, email-notification
+teampulseid:
+ticketid: 1111828, 1072170, 1060801, 1054819, 1087785, 1100004, 1058035, 1087785
+pitsid:
+
+---
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>{{site.tp}}, {{site.bs}} {{site.bs-n}}</td>
+  </tr>
+</table>
+
+## Description
+After changing the application name in Telerik Platform portal, the old name {{Application.Title}} still appearing in all automated emails.
+
+## Cause
+Internal limitation. {{Application.Title}} gets its value not from the application name but from another field (backend project name) which is not changed when you change the appliaction name from the portal UI.
+
+## Solution
+At the moment, you can change the backend project name only programmatically.
+
+- Locate the app in Telerik Platform and the App ID and Master Key as explained [here](http://docs.telerik.com/platform/backend-services/javascript/security/security-keys-get)

```

Comment:
Again a suggestion for different wording:
Locate the app in Telerik Platform and then obtain its App ID and Master Key as explained...